onnxruntime/test/python/quantization/test_save_activations.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
+# coding: utf-8
```

Comment:
```suggestion
```

no need for the shebang if it is not intended as an executable. the coding hint is no longer needed